docs/_build/triggers/hook-trigger.md

```diff
@@ -2,7 +2,8 @@
 title: Add a REST Hook trigger
 order: 3
 layout: post-toc
-redirect_from: 
+redirect_from: /build/resthooks
```

Comment:
I tried testing the different redirect here, but /build/resthooks still goes to /build/cli-hook-trigger. I believe it's supposed to go to /build/hook-trigger right?

I'm not sure how the URL is constructed, it's possible there's some logic stored elsewhere that needs to be changed for the redirects.

Reply:
Troubleshooted with Marina this might have to do with my local setup, this actually works fine. So will approve then.